pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -349,25 +349,29 @@ func (f *framework) RunPostFilterPlugins(
 	return nil
 }
 
-// RunScorePlugins runs the set of configured scoring plugins. It returns a map that
+// RunScorePlugins runs the set of configured scoring plugins. It returns a list that
 // stores for each scoring plugin name the corresponding NodeScoreList(s).
 // It also returns *Status, which is set to non-success if any of the plugins returns
 // a non-success status.
-func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.Node) (PluginToNodeScoreMap, *Status) {
-	pluginToNodeScoreMap := make(PluginToNodeScoreMap, len(f.scorePlugins))
+func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.Node) (PluginToNodeScores, *Status) {
+	pluginToNodeScoreList := make(PluginToNodeScores, len(f.scorePlugins))
```

Comment:
change the variable name to `pluginToNodeScores`

Reply:
Done